capture_service/trace_mgr.h

```diff
@@ -66,6 +66,8 @@ class AndroidTraceManager : public TraceManager
     virtual void WaitForTraceDone() override;
 
 private:
+    void               TraceByFrame();
+    void               TraceByDuration();
     bool               ShouldStartTrace();
```

Comment:
not related to this change, but should "ShouldStartTrace()" and "ShouldStopTrace()" both be `const` since they don't change any member?